### PR TITLE
Add note from borg with-lock about copy repo

### PR DESCRIPTION
**Pull request by infectormp**
I think this will more helpful to have all info about copy repo in one section

### CODE REVIEW

File: docs/faq.rst [hunk 1]
```diff
@@ -43,6 +43,13 @@ If you want to have 2 independent backups, it is better to do it like this:
 - client machine ---borg create---> repo1
 - client machine ---borg create---> repo2
 
+.. note::
+
+    If you copy a repository with the lock held, the lock will be present in
+    the copy. Thus, before using borg on the copy from a different host,
+    you need to use :ref:`with-lock` on the copied repository, because
+    Borg is cautious and does not automatically remove stale locks made by a different host.
+
```
Comment (ThomasWaldmann) [inline docs/faq.rst]: http://borgbackup.readthedocs.io/en/stable/usage/lock.html

there is already a similar not there, i do not think we should duplicate it (but rather improve it at one place and link to that place where needed).

File: docs/faq.rst [hunk 2]
```diff
@@ -27,7 +27,7 @@ Can I copy or synchronize my repo to another location?
 ------------------------------------------------------
 
 Yes, you could just copy all the files. Make sure you do that while no
-backup is running (use `borg with-lock ...`). So what you get here is this:
+backup is running (use :ref:`with-lock` to prevent this). So what you get here is this:
```
Comment (ThomasWaldmann) [inline docs/faq.rst]: remove trailing blank please

**Comment by infectormp** [comment]
what do you prefer move notice from `borg with-lock` to faq section or just make link from faq to `borg with-lock`

**Comment by ThomasWaldmann** [comment]
@infectormp improve at / move to docs location, link from faq location.

File: docs/faq.rst [hunk 3]
```diff
@@ -27,7 +27,13 @@ Can I copy or synchronize my repo to another location?
 ------------------------------------------------------
 
 Yes, you could just copy all the files. Make sure you do that while no
-backup is running (use `borg with-lock ...`). So what you get here is this:
+backup is running. If you copy a repository while backup is running 
+lock held will be present in the copy. Thus, before using borg on the copy
```
Comment (ThomasWaldmann) [inline docs/faq.rst]: If you copy a repository while a backup is running, (the lock held | a lock) will be present in the copy.

**Pull request merged**